etc/udiskie.profile

```diff
@@ -0,0 +1,35 @@
+# Firejail profile for udiskie
+# Description: Removable disk automounter using udisks
+# This file is overwritten after every install/update
+# quiet
+# Persistent local customizations
+include udiskie.local
+# Persistent global definitions
+include globals.local
+
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc include disable-programs.inc
+include disable-xdg.inc
+
+caps.drop all
+machine-id
+net none
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+seccomp
+shell none
+tracelog
+
+private-cache
```

Comment:
What about `private-bin` and `private-etc`?

Reply:
> What about `private-bin` and `private-etc`?

this apparently does not hurt: ```private-etc ld.so.cache,ld.so.preload,ld.so.conf,ld.so.conf.d,locale,locale.alias,locale.conf,localtime,alternatives,mime.types,xdg```

Reply:
No `fonts`?